src/main/java/org/apache/maven/it/Verifier.java

```diff
@@ -103,23 +103,32 @@
 
     private boolean debug;
 
+    /** 
+     * If {@code true} uses {@link ForkedLauncher}, if {@code false} uses {@link Embedded3xLauncher},
+     * otherwise considers the value {@link #forkMode}.
+     */
     private Boolean forkJvm;
 
     private String logFileName = LOG_FILENAME;
 
-    private String defaultMavenHome;
-
-    private String defaultClassworldConf;
-
-    private String defaultClasspath;
+    private String mavenHome;
 
     // will launch mvn with --debug 
     private boolean mavenDebug = false;
 
+    /**
+     * Either "auto" (use {@link ForkedLauncher} when {@link #environmentVariables} is not empty,
+     * otherwise use {@link Embedded3xLauncher}) , "embedder" (always use {@link Embedded3xLauncher})
+     * or something else (always use {@link ForkedLauncher}).
+     * Set through system property {@code verifier.forkMode}.
+     * Only relevant if {@link #forkJvm} is {@code null}.
+     */
     private String forkMode;
 
     private boolean debugJvm = false;
 
+    private boolean useWrapper;
```

Comment:
This

Reply:
Obviously when using a dedicated maven home no `mvnw` should be used, therefore this required to determine this in the constructor and introducing this field.